### ISSUE TITLE
sp_Blitz, sp_BlitzFirst: add check for recent service restart

### ISSUE THREAD

**Issue by BrentOzar** [BrentOzarULTD/SQL-Server-First-Responder-Kit#1484]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
In Azure Managed Instances, failovers are pretty quick, and you may not notice that you've failed over. This affects wait stats, index stats, troubleshooting, etc. However, we don't have a quick way of telling people, and it should be kinda high priority.

**What is the expected behavior?**
Add a priority 10 check to warn if the instance was recently restarted. (Doesn't need to be Azure MI specific.)

**Comment by BlitzErik** [comment]
What do you consider recent? We have a server/service restart checks. We could just promote one of them to a higher priority if it's within [your window here].

**Comment by BrentOzar** [comment]
Yeah, I think we just say the date right now when CheckServerInfo = 1, but we don't have an explicit "the service was restarted within the last X hours" check.

**Comment by BlitzErik** [comment]
Yessireebob! How many hours back do we care about for a high level warning? 24?

**Comment by BrentOzar** [comment]
Yeah, I think that makes sense.
